core/client/app/mixins/editor-base-controller.js

```diff
@@ -211,12 +211,13 @@ export default Ember.Mixin.create({
 
     showSaveNotification: function (prevStatus, status, delay) {
         var message = this.messageMap.success.post[prevStatus][status],
-            path = this.get('model.absoluteUrl'),
             type = this.get('postOrPage'),
             notifications = this.get('notifications');
 
         if (status === 'published') {
-            message += `&nbsp;<a href="${path}">View ${type}</a>`;
+            message += `&nbsp;<a href="${this.get('model.absoluteUrl')}">View ${type}</a>`;
+        } else {
+            message += `&nbsp;<a href="${this.get('model.previewUrl')}">Preview ${type}</a>`;
```

Comment:
IMO as it's a preview it would make sense, so that someone could preview it and then immediately go back to writing, without using the back button of the browser.